### ISSUE TITLE
[Package Issue]: Cloudflare.Warp Installer hash does not match!

### ISSUE THREAD

**Issue by isanjaymenon** [microsoft/winget-pkgs#82072]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installer hash mismatch.

### Brief description of your issue

When installing Cloudflare WARP, I get "Installer hash does not match error".

### Steps to reproduce

1. Open Windows Terminal
2. Type this ```winget install --id=Cloudflare.Warp -e```

### Actual behavior

PS C:\Users\Sanjaymenon> winget install --id=Cloudflare.Warp -e
Found Cloudflare WARP [Cloudflare.Warp] Version 22.8.857.0
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://1111-releases.cloudflareclient.com/windows/Cloudflare_WARP_Release-x64.msi
  99.1 MB
Installer hash does not match; to override this check use --force

### Expected behavior

Successfully verified installer hash
Starting package install...
Successfully installed

### Environment

```shell
[winget --info]
Windows Package Manager v1.2.10691
Windows: Windows.Desktop v10.0.22621.608
Package: Microsoft.DesktopAppInstaller v1.17.10691.0
```

### Screenshots and Logs

![Screenshot 2022-10-02 141139](https://user-images.githubusercontent.com/83574788/193445725-e58ad941-0e00-4b33-b8d1-0c754bc4efac.png)

**Comment by mdanish-kh** [comment]
- https://github.com/microsoft/winget-pkgs/pull/81997
 
Will be resolved once this PR gets merged.

**Comment by OfficialEsco** [comment]
close with reason: resolved in https://github.com/microsoft/winget-pkgs/pull/81997 ;

**Comment by isanjaymenon** [comment]
Resolved in #81997

![Screenshot 2022-10-02 165944](https://user-images.githubusercontent.com/83574788/193451693-2464891c-959a-4996-9fd7-871b991949b0.png)
